### ISSUE TITLE
wrong compression: bzip2 is wrong, should be bz2

### ISSUE THREAD

**Issue by m1ci** [dbpedia/databus#162]
In list for https://databus.dbpedia.org/dbpedia/mappings/mappingbased-objects-uncleaned we have 
https://databus.dbpedia.org/dbpedia/mappings/mappingbased-objects-uncleaned/2022.09.01/mappingbased-objects-uncleaned_lang=en.ttl.bzip2
and correct should be bz2

**Comment by JJ-Author** [comment]
thats not incorrect. the databus file identifiers are independent of the actual filename. the file id correctly redirects to the filename with bz2 file suffix.
the question is what happens to the id generation if an bz2 and a bzip2 file are registered with the exact same content variants.
there might be clash of identifierst than. you might wanna check that actually.

**Comment by JJ-Author** [comment]
a little related to this https://github.com/dbpedia/databus/issues/175
might have been introduced with the migration script?
